docs/modules/user/pages/quickstart.adoc

```diff
@@ -58,6 +58,46 @@ services:
     restart: unless-stopped
 ....
 
+Podman quadlet:
+
+[source,ini]
+....
+[Unit]
+Description=Podman Wolf Gamestreaming
+
+[Service]
+TimeoutStartSec=900
+ExecStartPre=-/usr/bin/mkdir /tmp/sockets
+ExecStartPre=-/usr/bin/podman rm --force WolfPulseAudio
+Restart=on-failure
+RestartSec=5
+StartLimitBurst=5
+
+[Container]
+AutoUpdate=registry
+ContainerName=%N
+HostName=%N
+Image=ghcr.io/games-on-whales/wolf:stable
+AddCapability=CAP_SYS_PTRACE
+AddCapability=CAP_NET_ADMIN
+Network=host
+SecurityLabelDisable=true
+PodmanArgs=--ipc=host --device-cgroup-rule "c 13:* rmw"
+AddDevice=/dev/dri
+AddDevice=/dev/uinput
+AddDevice=/dev/uhid
+Environment=HOST_APPS_STATE_FOLDER=/etc/wolf
+Environment=GST_DEBUG=2
```

Comment:
Any reason why you manually specified those? It shouldn't be required since it's the same as the defaults  https://github.com/games-on-whales/wolf/blob/e10d00bba435285279fa013cad320afba97f362d/docker/wolf.Dockerfile#L115-L116

Reply:
The original quadlet file that was posted on discord set a bunch more environment variables which I removed.
Apparently I forgot to remove the `GST_DEBUG` line.
The `HOST_APPS_STATE_FOLDER` I kept because both the docker and docker-compose blocks set that variable as well.

If you want I can remove it from all of them...